Update README (#20139)
airbytehq/airbyte: README.md

<p align="center">
  <a href="https://airbyte.com"><img src="https://assets.website-files.com/605e01bc25f7e19a82e74788/624d9c4a375a55100be6b257_Airbyte_logo_color_dark.svg" alt="Airbyte"></a>
</p>
<p align="center">
    <em>Data integration platform for ELT pipelines from APIs, databases & files to databases, warehouses & lakes</em>
</p>
<p align="center">
<a href="https://github.com/airbytehq/airbyte/stargazers/" target="_blank">
    <img src="https://img.shields.io/github/stars/airbytehq/airbyte?style=social&label=Star&maxAge=2592000" alt="Test">
</a>
<a href="https://github.com/airbytehq/airbyte/releases" target="_blank">
    <img src="https://img.shields.io/github/v/release/airbytehq/airbyte?color=white" alt="Release">
</a>

> <a href="https://airbytehq.slack.com/" target="_blank">
>     <img src="https://img.shields.io/badge/slack-join-white.svg?logo=slack" alt="Slack">
> </a>

Reviewer: ```suggestion
<a href="https://airbytehq.slack.com/" target="_blank">
    <img src="https://img.shields.io/badge/slack-join-white.svg?logo=slack" alt="Slack">
</a>
<a href="https://www.youtube.com/c/AirbyteHQ/?sub_confirmation=1" target="_blank">
    <img alt="YouTube Channel Views" src="https://img.shields.io/youtube/channel/views/UCQ_JWEFzs1_INqdhIO3kmrw?style=social">
</a>
```

<a href="https://github.com/airbytehq/airbyte/actions/workflows/gradle.yml" target="_blank">
    <img src="https://img.shields.io/github/workflow/status/airbytehq/airbyte/Airbyte%20CI" alt="Build">
</a>
<a href="https://github.com/airbytehq/airbyte/tree/a9b1c6c0420550ad5069aca66c295223e0d05e27/LICENSE/README.md" target="_blank">
    <img src="https://img.shields.io/static/v1?label=license&message=MIT&color=white" alt="License">
</a>
<a href="https://github.com/airbytehq/airbyte/tree/a9b1c6c0420550ad5069aca66c295223e0d05e27/LICENSE/README.md" target="_blank">
    <img src="https://img.shields.io/static/v1?label=license&message=ELv2&color=white" alt="License">
</a>
</p>

> We believe that only an open-source solution to data movement can cover the long-tail of connectors while empowering data engineers to customize existing connectors. Airbyte connectors take the form of a Docker image which follows the Airbyte specification and can be implemented in any language. You can replicate data from any source to any destination. We provide a low-code Conector Development Kit (CDK) for API connectors, a Python CDK and Java templates to quickly create new connectors.

Reviewer: ```suggestion
We believe that only an **open-source** solution to data movement can cover the **long tail of data sources** while empowering data engineers to **customize existing connectors**. Our ultimate vision is to help you move data from any source to any destination. Airbyte connectors can take the form of:

 - a [low-code Connector Development Kit](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview) (CDK) for API connectors ([demo](https://www.youtube.com/watch?v=i7VSL2bDvmw))
 - a [Python Connector Development Kit](https://docs.airbyte.com/connector-development/cdk-python/)  ([tutorial](https://docs.airbyte.com/connector-development/tutorials/cdk-speedrun))
 - a Docker image which follows [the Airbyte specification](https://docs.airbyte.com/understanding-airbyte/airbyte-protocol/) and can be implemented in _any_ language. 
 - There are community examples for JavaScript and Java as well.
```

> Airbyte comes with a built-in scheduler and Temporal to orchestrate workers. Airbyte normalizes extracted data with dbt models and can trigger custom transformation in SQL and dbt. You can also orchestrate Airbyte syncs with Airflow, Prefect and Dagster.

Reviewer: ```suggestion
Airbyte comes with a built-in scheduler and [Temporal](https://airbyte.com/blog/scale-workflow-orchestration-with-temporal) to orchestrate workers and ensure reliability at scale. Airbyte normalizes extracted data with dbt models ([talk](https://www.youtube.com/watch?v=saXwh6SpeHA)) and can trigger custom transformation in SQL and dbt ([docs](https://docs.airbyte.com/operator-guides/transformation-and-normalization/transformations-with-sql)). You can also orchestrate Airbyte syncs with [Airflow](https://docs.airbyte.com/operator-guides/using-the-airflow-airbyte-operator), [Prefect](https://docs.airbyte.com/operator-guides/using-prefect-task) or [Dagster](https://docs.airbyte.com/operator-guides/using-dagster-integration).
```

> ![Airbyte OSS Connections UI](https://user-images.githubusercontent.com/2302748/205949986-5207ca24-f1f0-41b1-97e1-a0745a0de55a.png)

Reviewer: ```suggestion
![Airbyte OSS Connections UI](https://user-images.githubusercontent.com/2302748/205949986-5207ca24-f1f0-41b1-97e1-a0745a0de55a.png) `#000`
```

Check [300+ connectors](https://docs.airbyte.com/integrations/) for APIs, databases, warehouses and lakes.

## Quick start

### Run Airbyte locally

You can run Airbyte locally with Docker.

```bash

> git clone https://github.com/airbytehq/airbyte.git

Reviewer: ```suggestion
git clone --depth 1 https://github.com/airbytehq/airbyte.git
```
